.github/workflows/node.js.yml

```diff
@@ -16,14 +16,14 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
+        node-version: [14.x, 16.x, 18.x, 19.x]
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
```

Comment:
I think this can cause error - eg: during `npm i` `styled-components` get updated and CSS breaks. - we experienced this in the past ?

Reply:
Good catch. Reverting to `npm install`